aion/web/elm/src/Update.elm

```diff
@@ -121,32 +140,32 @@ update msg model =
             else
                 model ! []
 
-        NoOperation ->
-            model ! []
-
-        SetNewQuestionContent questionContent ->
+        UpdateCreateQuestionForm name value ->
             let
                 oldPanelData =
                     model.panelData
             in
-                { model | panelData = { oldPanelData | newQuestionContent = questionContent } } ! []
+                { model
+                    | panelData =
+                        { oldPanelData | createQuestionForm = Forms.updateFormInput oldPanelData.createQuestionForm name value }
+                }
+                    ! []
 
-        SetNewAnswerContent answerContent ->
+        CreateNewQuestionWithAnswers ->
             let
-                oldPanelData =
-                    model.panelData
+                possibleFields =
+                    [ "question", "answers", "subject" ]
```

Comment:
I totally wouldn't hardcode these (definitely not in **Update.elm**, I say we stick to only having logic here, not data). Maybe we could find a cozy place for constants like that? We could put it in appropriate Models.elm or Constants.elm file.